src/store/metadata/oxigraph/store.py

```diff
@@ -219,6 +220,37 @@ def get_versions(
         Gets all versions of a specific edition of a specific dataset
         """
 
+        # Populate the graph from the database
+        graph = self.db
+
+        # Use the construct wrappers to pull the raw RDF triples
+        # (as one rdflib.Graph() for each function) and add them
+        # together to create a single Graph of the
+        # data we need.
+        result: Graph = construct_versions(graph, dataset_id, edition_id)
+
+        # Serialize the graph into jsonld
+        data = json.loads(result.serialize(format="json-ld"))
+
+        # Use a context file to shape our jsonld, removing long form references
+        data = jsonld.flatten(
+            data, {"@context": constants.CONTEXT, "@type": "hydra:Collection"}
+        )
+        versions_graph = _get_single_graph_for_field(data, "@type")
+        if versions_graph is None:
+            return None
+
+        # TODO Fix context weirdness - at the moment, the flatten() method is changing @type to `versions_url` and `editions` to `versions`
+        versions_graph["@type"] = "hydra:Collection"
+        versions_graph["versions"] = versions_graph.pop("versions")
```

Comment:
this isn't doing anything, we're effectively renaming "versions" to "versions".

the code you were basing this off has an issue where flatten was naming the "editions" fields as "versions" which doesnt apply here - you can remove the TODO for the same reason.